operator/api/v1alpha1/securedcluster_types.go

```diff
@@ -104,11 +104,22 @@ type SecuredClusterSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Custom Default Image Registry",order=14,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced"}
 	RegistryOverride string `json:"registryOverride,omitempty"`
 
+	// Controls if process baselines are locked once the observation period ends
```

Comment:
```suggestion
	// Settings related to process baseline feature.
```
Or something similar, probably best to call it the same as you refer to this whole feature in docs.